R/run_pipelines.R

```diff
@@ -65,6 +65,35 @@ run_pipelines <- function(path = choose_directory(),
 
   }
 
+  #Determine operating system
+  OS <- tolower(sessionInfo()$running)
+
+  #Drop populations from Access-based primary data if running on Mac
+  PopID_Access <- c("HOG", "OOS", "VLI", "BUU", "LIE", "WAR", "WES", "AMM")
+
+  if(grepl(pattern = 'mac', x = OS)){
+
+    if(length(PopID[which(PopID%in%PopID_Access)] > 0)){
+      warning(paste0('Pipelines not run for the following populations due to OS incopatibility: ',
+                    toString(PopID[which(PopID%in%PopID_Access)]),
+                    ". To obtain standard format data for these populations, please run on a Windows OS.")
+              )
+    }
+
+    PopID <- PopID[which(!(PopID%in%PopID_Access))]
+
+  }else if(grepl(pattern = 'windows', x = OS)){
+
+    PopID <- PopID
+
```

Comment:
This is redundant right? PopID is already defined before the ifelse, so no need to redefine it here.
```suggestion
```